documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -508,6 +508,18 @@ Empty for standalone servers.
 ** `status`  HEALTHY,UNHEALTHY or REBALANCING
 ** `cache_name` the name of the cache as defined in the configuration.
 
+[[rest_v2_cache_manager_health_status]]
+==== Health Status
+
+To simply obtain the health status of the cache managers, without the need to go through authentication, do a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/cache-managers/{cacheManagerName}/health/status
+----
+
+The response will be in `text/plain` with `HEALTHY`,`REBALANCING` or `UNHEALTHY`.
```

Comment:
If using this for a load balancer health check, where the use of substring matching is common, I'd probably rename "REBALANCING" to "HEALTHY REBALANCING" (because it is still healthy from the pov of a user), and "UNHEALTHY" to "DEGRADED" (so that it doesn't contain the string)

Reply:
Sounds good to me, @ryanemerson, thoughts?

Reply:
Makes sense to me, it'll make the probe check simpler as well.

Reply:
done